### PR TITLE
Refactor CSS Classes by Using row, col and generic-style CSS Classes.

### PR DESCRIPTION
**Pull request by gtarique**

### CODE REVIEW

**Comment by becky-intelletive** [comment]
Looking good! Could you include some before/after screenshots of a few pages? In several sizes? We want to make sure that the appearance isn't changing, both on desktop and mobile.

**Comment by gtarique** [comment]
> Looking good! Could you include some before/after screenshots of a few pages? In several sizes? We want to make sure that the appearance isn't changing, both on desktop and mobile.

I have taken screenshots of pages both on desktop and mobile. You will find the screenshots [here](https://drive.google.com/drive/folders/1kfatIcVqQ8CrRRi6fgWe77xEEjpqOxbb?usp=drive_link).

File: dash/frontend/src/generic-styles.scss [hunk 1]
```diff
@@ -65,9 +67,16 @@ $classesToGenerate: (
     values: (
       auto: auto, unset: unset,
       fit: 'fit-content',
-      0: 0, 50: 50%, 75: 75%, 100: 100%,
+      0: 0, 50: 50%, 75: 75%, 97: 97%, 100: 100%,
```
Comment (becky-intelletive) [inline dash/frontend/src/generic-styles.scss:70]: Is this value in use anymore? (I didn't see it but maybe I just missed it.)

File: dash/frontend/src/generic-styles.scss [hunk 2]
```diff
@@ -116,17 +131,44 @@ $classesToGenerate: (
...
+    abbreviation: 'overflow-x',
+    values: (
+      auto: 'auto',
+      visible: 'visible',
+      scroll: 'scroll',
+      hidden: 'hidden'
+    )
+  ),
+  background-color: (
+  abbreviation: 'background-color',
```
Comment (becky-intelletive) [inline dash/frontend/src/generic-styles.scss:154]: this needs the indent

File: dash/frontend/src/generic-styles.scss [hunk 3]
```diff
@@ -116,17 +131,44 @@ $classesToGenerate: (
...
+  overflow-x: (
+    abbreviation: 'overflow-x',
+    values: (
+      auto: 'auto',
+      visible: 'visible',
+      scroll: 'scroll',
+      hidden: 'hidden'
+    )
+  ),
+  background-color: (
```
Comment (becky-intelletive) [inline dash/frontend/src/generic-styles.scss:153]: Is this being used? I didn't see it in use, but maybe I missed it.

File: dash/frontend/src/generic-styles.scss [hunk 4]
```diff
@@ -116,17 +131,44 @@ $classesToGenerate: (
...
+  overflow: (
+    abbreviation: 'overflow',
+    values: (
+      auto: 'auto',
+      visible: 'visible',
+      scroll: 'scroll',
+      hidden: 'hidden'
+    )
+  ),
+  overflow-x: (
```
Comment (becky-intelletive) [inline dash/frontend/src/generic-styles.scss:144]: Is this being used? I didn't see it in use, but maybe I missed it.

File: dash/frontend/src/app/modules/private/pages/cluster/kubernetes-namespaces/kubernetes-namespaces.component.html [hunk 5]
```diff
@@ -1,5 +1,5 @@
-<div class="page-container">
-  <div class="h-xs-fit page-container-content">
+<div class="row display-xs-block h-xs-100 overflow-xs-auto mr-xs-1">
+  <div class="col-xs ml-xs-6 mt-xs-6">
```
Comment (becky-intelletive) [inline dash/frontend/src/app/modules/private/pages/cluster/kubernetes-namespaces/kubernetes-namespaces.component.html:2]: `ml-xs-6`
is the same as
`ms-xs-6`
in this case

File: dash/frontend/src/app/modules/private/pages/cluster/kubernetes-namespaces/kubernetes-namespaces.component.html [hunk 6]
```diff
@@ -1,5 +1,5 @@
-<div class="page-container">
-  <div class="h-xs-fit page-container-content">
+<div class="row display-xs-block h-xs-100 overflow-xs-auto mr-xs-1">
```
Comment (becky-intelletive) [inline dash/frontend/src/app/modules/private/pages/cluster/kubernetes-namespaces/kubernetes-namespaces.component.html:1]: is
`mr-xs-1`
the same as
`me-xs-1`?

File: dash/frontend/src/generic-styles.scss (hunk 3 above)
Comment (gtarique) [reply dash/frontend/src/generic-styles.scss:153]: @becky-intelletive  we have used this in cluster list and image scan result pages

File: dash/frontend/src/generic-styles.scss (hunk 1 above)
Comment (gtarique) [reply dash/frontend/src/generic-styles.scss]: @becky-intelletive  We have just removed this value.

File: dash/frontend/src/app/modules/private/pages/cluster/kubernetes-namespaces/kubernetes-namespaces.component.html [hunk 7]
```diff
@@ -1,5 +1,5 @@
-<div class="page-container">
-  <div class="h-xs-fit page-container-content">
```
[the remark this answers is not in the document]
Comment (gtarique) [reply dash/frontend/src/app/modules/private/pages/cluster/kubernetes-namespaces/kubernetes-namespaces.component.html:2]: @becky-intelletive you are right, its needed and restored.

File: dash/frontend/src/generic-styles.scss (hunk 4 above)
Comment (gtarique) [reply dash/frontend/src/generic-styles.scss:144]: @becky-intelletive we have used overflow-x-xs-auto class in multiple pages.

File: dash/frontend/src/app/modules/private/pages/cluster/kubernetes-namespaces/kubernetes-namespaces.component.html (hunk 6 above)
Comment (gtarique) [reply dash/frontend/src/app/modules/private/pages/cluster/kubernetes-namespaces/kubernetes-namespaces.component.html]: @becky-intelletive  yes. we have just updated mr-xs-1 to me-xs-1.

**Review by becky-intelletive — APPROVED** [review]

**Comment by becky-intelletive** [comment]
@gtarique - please pull in the latest main branch to fix the selenium test suite :)

**Comment by gtarique** [comment]
@becky-intelletive I have pulled the latest main branch. thanks.

**Pull request merged**